lms/static/styles/moodle_pages/moodle_pages.scss

```diff
@@ -0,0 +1,249 @@
+// This stylesheet contains a re-implementation of some of Moodle styles for
+// pages, so that Moodle pages served through our LMS app + Via look close to
+// how the page looks in Moodle.
+//
+// The markup generated for pages does not include specific classes, so we only
+// need to define styles for HTML elements.
+//
+// The styles defined here are mainly extracted from the next Moodle files:
+// * Base definition:
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/bootstrap/_reboot.scss
+// * Variables:
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/preset/default.scss
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/bootstrap/_variables.scss
+
+// These are some of the sass variables defined in the files mentioned above:
+$font-family-sans-serif: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+  'Helvetica Neue', Arial, 'Noto Sans', 'Liberation Sans', sans-serif,
+  'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+$font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas,
+  'Liberation Mono', 'Courier New', monospace;
+$font-size-base: 0.9375rem;
+$gray-600: #6a737b;
+$gray-900: #1d2125;
+$link-color: #ec7f13;
+$link-hover-color: darken($link-color, 15%);
+
+body {
+  font-family: $font-family-sans-serif;
+  font-size: $font-size-base;
+  line-height: 1.5;
+  color: $gray-900;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+ol,
+ul,
+dl {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+ol ol,
+ul ul,
+ol ul,
+ul ol {
+  margin-bottom: 0;
+}
+
+dt {
+  font-weight: 700;
+}
+
+dd {
+  margin-bottom: 0.5rem;
+  margin-left: 0;
+}
+
+blockquote {
+  margin: 0 0 1rem;
+}
+
+b,
+strong {
+  font-weight: bolder;
```

Comment:
TIL that `bolder` is a valid font weight.

Reply:
If you want an even bolder one, you can set `boldest` 😝